docs/tutorial/part-two/index.md

```diff
@@ -229,8 +230,8 @@ Ah, this is starting to look nice!
 What we're seeing here is the default CSS Typography.js produces. We can easily
 customize it, however. Let's do that.
 
-In your site, create a new directory at `src/utils`. There create a file named
-`typography.js`. In it, add the following code.
+In your site, create a new directory at `src/utils` (hint: in code editors, creating a new directory looks like creating a new folder). In that directory, create a file named
```

Comment:
a directory is the same thing as a folder :-)

If this is confusing, let's make sure to use the same word throughout the tutorial (and documentation).

Reply:
I believe directory is more standard in this industry.  "folder" seems so non-professional.